vivo-project/Vitro
# Vivo 1630 Year field keeps the same label even when language is changed

**[JIRA Issue](https://jira.duraspace.org/projects/VIVO)**: https://jira.duraspace.org/browse/VIVO-1630


# What does this pull request do?
Use of localized Year from property file.
[Editors note] - 'Year' is hard-coded in english. It will appear as 'Year' regardless of the language selected.

# How should this be tested?
edit a person postion with a data interval
[Editors note] - This pull request creates an i18n value for 'Year.' Test the problem by selecting a different language than English and 'Year' will remain. After implementing the change (and the associated VIVO-languages pull request) 'Year' should change with the language. 


# Additional Notes:
i have VIVO-1649 as prerequsite to start my instance on blazegraph

# Interested parties
@VIVO-project/vivo-committers
